README.markdown

```diff
@@ -236,12 +236,21 @@ func runUnsafe(unsafe string) {
 
     vm := otto.New()
     vm.Interrupt = make(chan func(), 1) // The buffer prevents blocking
+	watchdogCleanup := make(chan bool)
+	defer func() {
```

Comment:
No need for a bool channel and a send, instead you can use a struct{} channel and simply close it.

Reply:
apologies for the excessively slow response here: resolved via 675d0d0d7f62507858e48f70f68c1ca75c2ccf62